src/mpcCoreKit.ts

```diff
@@ -1190,7 +1189,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
   }
 
   private _getSignatures(sessionData: TorusKey["sessionData"]["sessionTokenData"]): string[] {
-    return sessionData.map((session) => JSON.stringify({ data: session.token, sig: session.signature }));
+    return sessionData.filter((session) => !!session).map((session) => JSON.stringify({ data: session.token, sig: session.signature }));
```

Comment:
so, now it is possible that the sessionData item is undefined ?

Reply:
<img width="945" alt="Screenshot 2024-10-02 at 4 06 36 PM" src="https://github.com/user-attachments/assets/1a929768-9ea4-486b-900c-d62ed23d3518">

So there is a check in torus.js which pushes undefined to session data in case that particular node call fails. Before returning we are not filtering out undefined vals in torus.js

ideally the fix should be done in torus.js which i will do. But i need to check in other places where these sigs are being used to make sure changing torus.js wont causing anything to break and also torus.js fix will have to updated in custom auth thn sfa and thn tkey and thn mpc core kit, so i m just filtering it here first and if i see this will fix the issue , thn i will backport this change to torus.js sdk.

Reply:
maybe adding a comment to the code that explains this briefly would help. thx for explaining here.